data/saas/dataset/mixpanel_dataset.yml

```diff
@@ -0,0 +1,140 @@
+dataset:
+  - fides_key: mixpanel_connector_example
+    name: Mixpanel Dataset
+    description: A sample dataset representing the Mixpanel connector for Fidesops
+    collections:
+      - name: user
+        fields:
+          - name: $distinct_id
+            data_categories: [user.derived.identifiable.unique_id]
+            fidesops_meta:
+              primary_key: True
+              # TODO - Custom data_type wasn't supported at the time it was written
```

Comment:
This might have been a mistake in the legacy implementation, the API documentation lists `unique_id` as a string ID https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelidentify so we can use `string` here instead.